### DISCUSSION TITLE
Split EBSD data based on orientation

### DISCUSSION THREAD

**Discussion by TomPotocny** [mtex-toolbox/mtex#1878] [Ask Anything]
Hello everyone
I am working on EBSD of Quartz pseudomorphs after Coesite. My data shows several generations of quartz with different orientations. Usually 2 or 3. How can I split my EBSD data based on orientation around ODF maxima? I would like to visualize them into a map and see in which part the group of specific oriented grains grew.

Thank you very much

TP
![ODFmaxima](https://github.com/mtex-toolbox/mtex/assets/137514671/dd3727fd-b5bd-44fe-ad8a-522dfa5775b3)
![phase](https://github.com/mtex-toolbox/mtex/assets/137514671/71c90333-e6e8-47f9-b1f1-c897c22b2e7b)

**Comment by kilir** [comment]
Hi,
have a look at odf/max(), e.g.
```matlab
mtexdata olivine
% create some non-indexed-Points
ebsdO = ebsd('o');
odf = calcDensity(ebsdO.orientations);
[~,opos] = odf.max('numlocal',3);
% for the 1st local maximum
cond = angle(ebsdO.orientations,opos(1)) < 30*degree;
plot(ebsdO(cond))

```
Cheers,
Rüdiger

**Reply by TomPotocny** [reply]
Dear Rüdiger,
thank you for your answer but I don't underestand how this should work
how can I calculate ODF from non indexed data without orientation


>> odf = calcDensity(ebsd0.orientations);
Error using phaseList/checkSinglePhase
There are no indexed data in this variable!

Tomas

**Reply by kilir** [reply]
The above was an example, you will most likely have to adapt the phase names.

Cheers,
Rüdiger

**Reply by TomPotocny** [reply]
Now I understand, 
Thank you very much

Cheers,

Tomas

**Reply by TomPotocny** [reply]
Dear Rüdiger,
Do you have any experience of why my calculated maximum does not visually match the ODF?
![MAX](https://github.com/mtex-toolbox/mtex/assets/137514671/5fa50886-28c4-4dd5-919a-b69cd2eb0133)

Thank you

**Reply by kilir** [reply]
Hi Tom,
that is because you do not look at an odf but at a pole figure. The maxima you get from an odf are orientations and different crystal directions are related by fixed rules - crystal symmetry elements - which of course they maintain when plotten in a pole figure. I.e. for a given odf maximum orientation, the resulting  11-20 direction need to be 90 degrees away from 0001. This is of course not required for the maxima in 0001 and 11-20 polefigures.

Hope that help.

Cheers,
Rüdiger

**Comment by TomPotocny** [comment]
> Hi, have a look at odf/max(), e.g.
> 
> ```matlab
> mtexdata olivine
> % create some non-indexed-Points
> ebsdO = ebsd('o');
> odf = calcDensity(ebsdO.orientations);
> [~,opos] = odf.max('numlocal',3);
> % for the 1st local maximum
> cond = angle(ebsdO.orientations,opos(1)) < 30*degree;
> plot(ebsdO(cond))
> ```
> 
> Cheers, Rüdiger
